benchmark/fs/bench-writeFileSync.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const common = require('../common.js');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bench = common.createBenchmark(main, {
+  encoding: ['utf8'],
+  hasFileDescriptor: ['true', 'false'],
+  stringLength: [1024, 4096, 65535],//, 1024 * 1024],
+  n: [1e3],
+});
+
+function main({ n, encoding, stringLength, hasFileDescriptor }) {
+  const enc = encoding === 'undefined' ? undefined : encoding;
+  const path = tmpdir.resolve(`.writefilesync-file-${process.pid}`);
+
+  const useFd = hasFileDescriptor === 'true';
+  const file = useFd ? fs.openSync(path, 'w') : path;
+
+  const data = 'a'.repeat(stringLength);
+
+  bench.start();
+  for (let i = 0; i < n; ++i) {
+    try {
+      fs.writeFileSync(file, data, enc);
+    } catch {
+      // do nothing
```

Comment:
Does this ever throw? We don't benchmark false paths in here. I don't we need this.